src/libstd/io/buffered.rs

```diff
@@ -322,7 +323,7 @@ impl<W> From<IntoInnerError<W>> for Error {
 }
 
 #[stable(feature = "rust1", since = "1.0.0")]
-impl<W: Send + fmt::Debug> error::Error for IntoInnerError<W> {
+impl<W: Reflect + Send + fmt::Debug> error::Error for IntoInnerError<W> {
```

Comment:
Could this and the bounds used below change from `Reflect` to `Any`? (just to reflect idiomatic usage)